src/FSharpLint.Core/Framework/HintParser.fs

```diff
@@ -783,22 +784,22 @@ module HintParser =
             satisfy isLetter
             .>> notFollowedBy (satisfy isLetter)
 
-        let ptuple (pparser:Parser<'T, unit>) : Parser<'T list, unit> =
+        let ptuple (pparser:Parser<'Type, unit>) : Parser<'Type list, unit> =
             skipChar '('
             >>. pparser
             .>> skipChar ','
             .>>. sepEndBy1 pparser (skipChar ',')
             .>> skipChar ')'
             |>> fun (func, rest) -> (func::rest)
 
-        let plist (pparser:Parser<'T, unit>): Parser<'T list, unit> =
+        let plist (pparser:Parser<'Type, unit>): Parser<'Type list, unit> =
             skipChar '['
             >>. spaces
             >>. sepEndBy pparser (skipChar ';')
             .>> spaces
             .>> skipChar ']'
 
-        let parray (pparser:Parser<'T, unit>): Parser<'T list, unit> =
+        let parray (pparser:Parser<'Type, unit>): Parser<'Type list, unit> =
```

Comment:
@Mersho replacing 'T with 'Type is like if you rename a variable called 'x' to 'variable'; please undo this change everywhere, and choose a better type name